apps/web/src/server/views/applications/case-s51/s51-successfully-unpublished.njk

```diff
@@ -23,15 +30,16 @@
 				html: bannerDetails
 			}) }}
 
-            <div class='govuk-!-font-size-19'>
-                <p class='govuk-!-margin-top-0 govuk-!-margin-right-2'>The S51 advice will be removed from the NI website within the hour.</p>
-            </div>
+			<div class='govuk-!-font-size-19'>
+				<p class='govuk-!-margin-top-0 govuk-!-margin-right-2'>The {% if isGeneralS51 %}General{% endif %} S51 advice will be removed from the NI website within the hour.</p>
```

Comment:
We probably want to check this text, perhaps with Chris B.  I know it's not really part of this story but I think it would have been a carry over from legacy NI:

"S51 advice will be removed from the NI website within the hour."

1) Is Front office known as NI website?
2) It used to take 1 hour but now is instant.